src/sandstorm/seccomp-bpf/gen-clean-h.c

```diff
@@ -0,0 +1,152 @@
+// This program prints out a "clean" header for use in bpf assembly, defining
+// constants we need from various system headers. `bpf_asm` will choke on the
+// originals, for two reasons:
+//
+// 1. They contain C code
```

Comment:
In the Workers Runtime, my solution to this was to have a special marker in the BPF file to mark where the code actually started. Then, I had an extra preprocessing step -- after running the C preprocessor, I use a `while read LINE` loop to in bash to remove everything up to the marker.